library/src/main/java/com/proxerme/library/connection/info/entity/EntryCore.java

```diff
@@ -0,0 +1,287 @@
+     */
+    protected EntryCore(Parcel in) {
+        id = in.readString();
+        name = in.readString();
+        genre = in.readString();
+        fsk = in.readString();
+        description = in.readString();
+        medium = in.readString();
+        count = in.readInt();
+        state = in.readInt();
+        rateSum = in.readInt();
+        rateCount = in.readInt();
+        clicks = in.readInt();
+        category = in.readString();
+    }
+
+    /**
+     * Returns the id of this entry.
+     *
+     * @return The id.
+     */
+    @NonNull
+    public String getId() {
+        return id;
+    }
+
+    /**
+     * Returns the name of this entry.
+     *
+     * @return the name.
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Returns the genre of this entry.
+     *
+     * @return The genre.
+     */
+    @NonNull
+    @GenreParameter.Genre
+    public String getGenre() {
+        return genre;
+    }
+
+    /**
+     * Returns an array of fsk names of this entry.
+     *
+     * @return An array of fsk names.
+     */
+    @NonNull
+    public String[] getFsk() {
+        return fsk.split(" ");
+    }
+
+    /**
+     * Returns the description of this entry.
+     *
+     * @return The description.
+     */
+    @NonNull
+    public String getDescription() {
+        return description;
+    }
+
+    /**
+     * Returns the medium of this entry.
+     *
+     * @return The medium.
+     */
+    @NonNull
+    @MediumParameter.Medium
+    public String getMedium() {
+        return medium;
+    }
+
+    /**
+     * Get the overall amount of episodes/chapters of this entry. It does include not available episodes.
+     *
+     * @return The amount of episodes/chapters.
+     */
+    @IntRange(from = 0)
+    public int getCount() {
+        return count;
+    }
+
+    /**
+     * Returns the view state for the current user of this entry. If no user is set then this is 0.
+     *
+     * @return The view state for the current user.
+     */
+    @IntRange(from = 0)
+    public int getState() {
+        return state;
+    }
+
+    /**
+     * Returns the sum of all ratings of this entry.
+     *
+     * @return The sum of ratings.
+     */
+    @IntRange(from = 0)
+    public int getRateSum() {
+        return rateSum;
+    }
+
+    /**
+     * Returns the amount of ratings of this entry.
+     *
+     * @return The amount of ratings.
+     */
+    @IntRange(from = 0)
+    public int getRateCount() {
+        return rateCount;
+    }
+
+    /**
+     * Returns the sum of all ratings of this entry.
+     *
+     * @return The sum of all ratings.
+     */
+    public int getRating() {
+        if (rateCount <= 0) {
+            return -1;
+        } else {
+            return rateSum / rateCount;
+        }
+    }
+
+    /**
+     * Returns the number of views of this entry. Will be deleted every 3 months.
+     *
+     * @return The number of views.
+     */
+    public int getClicks() {
```

Comment:
This comment was meant for the getClicks method, but yeah good idea.
I'll change that in the other classes with ratings, too.